crypto/dilithium/pqcrystals_dilithium_ref_common/poly.c

```diff
@@ -160,7 +160,7 @@ void poly_pointwise_montgomery(poly *c, const poly *a, const poly *b) {
   DBENCH_START();
 
   for(i = 0; i < N; ++i)
-    c->coeffs[i] = fqmul(a->coeffs[i], b->coeffs[i]);
+    c->coeffs[i] = fqmul((int64_t)a->coeffs[i], b->coeffs[i]);
```

Comment:
Why add this cast? fqmul takes a int32_t, which coeffs already is an array of.

Reply:
Thanks, removed in [3fc43e7](https://github.com/aws/aws-lc/pull/1894/commits/3fc43e7726dd0d3c0cbe697cb31651451b027113).